src/rust/ide/view/graph-editor/src/component/node/output/area.rs

```diff
@@ -276,8 +386,8 @@ impl Model {
 /// about this design decision, please read the docs for the [`node::Node`].
 #[derive(Clone,CloneRef,Debug)]
 pub struct Area {
-    pub frp : Frp,
-    model   : Rc<Model>,
+    pub frp   : Frp,
+    pub model : Rc<Model>,
```

Comment:
Does the model have to be public? Usually, all APIs should go through the FRP.

Reply:
very, very good point. Thanks for catching it.